### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by Lipt0nas**

### CODE REVIEW

File: docs/CONFIGURATION.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Configuration options
+
+It's possible to configure some settings by passing `--config` argument to the generator. All
+configuration keys are defined in `bindings.cpp` section.
+
+```bash
+uniffi-bindgen-cpp path/to/definitions.udl --config path/to/uniffi.toml
+```
+
+- `cdylib_name` - override the dynamic library name linked by generated bindings, excluding `lib`
```
Comment (arg0d) [inline docs/CONFIGURATION.md:10]: Does `cdylib_name` actually do anything?

Comment (Lipt0nas) [reply docs/CONFIGURATION.md:10]: It is stored and passed to the `BindingsConfig` implementation from uniffi, but otherwise it's not used internally

Comment (Lipt0nas) [reply docs/CONFIGURATION.md:10]: So it's used by uniffi if you're not using the `--library` option

Comment (Lipt0nas) [reply docs/CONFIGURATION.md]: After looking at it again, looks like i was wrong, guess this was left over from the start of the project and is in fact no being used at all, removed it from the code and the docs

Comment (arg0d) [reply docs/CONFIGURATION.md]: Its used by Kotlin/C# to specify dynamic library name directly in the bindings.

**Review by arg0d — APPROVED** [review]
looks good

**Pull request merged**